### PR TITLE
feature: remove delay param from notifyChange

### PR DESCRIPTION
**Pull request by peppsac**
Removed because it's a rarely used feature, and with 2 different use case, and both
can be implemented in a better way.

- reduce CPU usage (see Scheduler usage): iTowns partial update support cover this use-case,
with the benefit of not adding any latency.

- rescheduling an update after an error (see Layered...Processing.js): the user can use setTimeout
directly

### CODE REVIEW

File: src/Core/MainLoop.js [hunk 1]
```diff
@@ -58,6 +58,14 @@ MainLoop.prototype._update = function _update(view) {
 };
 
 MainLoop.prototype._step = function _step(view) {
+    const willRedraw = this.needsRedraw;
+
+    // Reset internal state before calling _update (so future calls to View.notifyChange()
+    // can properly change it)
+    this.needsRedraw = false;
+    this.renderingState = RENDERING_PAUSED;
+    view._changeSources.clear();
```
Comment (autra) [inline src/Core/MainLoop.js]: Why did you move these 2 lines? Should they be called *after* calling `this._update` below?

File: examples/planar.html [hunk 2]
```diff
@@ -158,7 +158,7 @@
                 view.camera.camera3D.updateMatrix();
                 view.camera.camera3D.updateMatrixWorld(true);
 
-                view.notifyChange(20, true);
+                view.notifyChange(true);
```
Comment (autra) [inline examples/planar.html:161]: You're not using setTimeout here, can you elaborate why it's not needed?

**Review by autra — CHANGES_REQUESTED** [review]

File: examples/planar.html (hunk 2 above)
Comment (peppsac) [reply examples/planar.html:161]: The delay was a left over from a test.
Here we want a continuous animation, so we want to keep the mainLoop running permanently.

File: src/Core/MainLoop.js (hunk 1 above)
Comment (peppsac) [reply src/Core/MainLoop.js]: Indeed I need to keep a copy of changeSources() before clearing it. Thanks :)

**Review by autra — APPROVED** [review]
LGTM, thanks :-)
